packages/@aws-cdk/aws-appmesh/lib/route-spec.ts

```diff
@@ -287,13 +294,449 @@ class HeaderMatchImpl extends HttpHeaderMatch {
 }
 
 /**
- * The criterion for determining a request match for this GatewayRoute
+ * Configuration for HTTP route match
+ */
+export interface HttpRouteMatchConfig {
+  /**
+   * Route CFN configuration for HTTP route match.
+   */
+  readonly requestMatch: CfnRoute.HttpRouteMatchProperty;
+}
+
+/**
+ * Defines HTTP route path or prefix request match.
+ */
+export abstract class HttpRoutePathOrPrefixMatch {
```

Comment:
I don't like this name.

I would rename this to `HttpRoutePathMatch` (I would also rename related types too, like `HttpRouteMatchConfig`, whose name right now doesn't really make much sense).

Reply:
Well noted.